index.html

```diff
@@ -61,5 +61,29 @@
 	
 	setupCanvas();
 
-	initSounds();
-</script>
\ No newline at end of file
+  initSounds();
+  
+var gameCanvas = document.getElementById("gameCanvas");
```

Comment:
var? :thinking:

Reply:
Btw, dont use ID's for selectors, you should use data-attributes instead.

Reply:
data attributes ? How exactly, could you give me an example ?